core/src/test/java/de/evoila/cf/broker/service/impl/AsyncDeploymentServiceImplTest/AsyncUpdateInstanceTest.java

```diff
@@ -2,75 +2,85 @@
 
 import de.evoila.cf.broker.exception.ServiceBrokerException;
 import de.evoila.cf.broker.model.JobProgress;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.when;
 
 class AsyncUpdateInstanceTest extends BaseTest {
 
-    private void whensForsyncUpdateInstanceThrowsException(JobProgress returnOfFailJob) throws ServiceBrokerException
-    {
+    private final static String JOB_PROGRESS_DESCRIPTION = "Updating service..";
+
+    private void whensForSyncUpdateInstanceThrowsException(Exception expectedException) throws ServiceBrokerException {
         mockSuccessfulStartJob(JobProgress.UPDATE);
-        ServiceBrokerException expectedException = new ServiceBrokerException("Test");
         when(deploymentService.syncUpdateInstance(serviceInstance, parameters, plan, platformService))
                 .thenThrow(expectedException);
-        when(startedJob.getId())
-                .thenReturn(JOB_PROGRESS_ID);
-        when(jobProgressService.failJob(eq(JOB_PROGRESS_ID), anyString()))
-                .thenReturn(returnOfFailJob);
     }
 
-    private void whensForSyncUpdateInstanceSucceeds(JobProgress returnOfSucceedProgress) throws ServiceBrokerException
-    {
+    private void whensForSyncUpdateInstanceThrowsServiceBrokerException() throws ServiceBrokerException {
```

Comment:
Are the methods whensForSyncUpdateInstanceThrowsServiceBrokerException and whensForSyncUpdateInstanceThrowsRuntimeException really needed? Why not call whensForSyncUpdateInstanceThrowsException directly?

Reply:
fixed